uber/templates/jobs/staffers.html

```diff
@@ -67,6 +69,26 @@
 </table>
 
 <br/> <br/>
+
+{% if c.HAS_ACCOUNTS_ACCESS %}
+    <div>
+    <button class="btn btn-default" onclick="checkAll()">Select All</button>
+    <button class="btn btn-default" onclick="uncheckAll()">Deselect All</button>
+    <h4>Accounts Selected: <i id="count">0</i></h4>
+    <form id="admin_settings" method="post" action="add_bulk_admin_accounts" onsubmit="return add_admin_list()" role="form" class="form-inline">
```

Comment:
plz to not `onsubmit="return add_admin_list()"`

You can hook into the form submission action with jQuery. Returning the function in the `onsubmit`...works...but it's not by any means convention.